pkg/trivy/executor_local.go

```diff
@@ -32,6 +33,8 @@ func NewLocalExecutor() *LocalExecutor {
 func (e LocalExecutor) Execute(_ context.Context, output io.Writer, args ...string) error {
 	shell := &exec.ShellExecutor{}
 
+	cmd := fmt.Sprintf("trivy %s", strings.Join(args, " "))
+
 	// We want to discard trivy logs, and only get the HTML output, that's why we set stderr to io.Discard
-	return shell.ExecuteWithWriters(output, io.Discard, e.Shell, "-c", strings.Join(args, " "))
```

Comment:
The comment above is not in sync with the code anymore. Either fix the comment or restore the `io.Discard` error output.
If I remember correctly the `io.Discard` helps reducing the noise of the output in case of errors, but might not be necessary with the Trivy command.

Reply:
I may have removed it to have some output for understanding the error. Restoring it.